docs/index.rst

```diff
@@ -1,44 +1,54 @@
 :orphan:
 
 ====================================
-ScyllaDB Open Source Documentation
+ScyllaDB Documentation
 ====================================
 
 .. meta::
-   :title: ScyllaDB Open Source Documentation
-   :description: ScyllaDB Open Source Documentation
-   :keywords: ScyllaDB Open Source, ScyllaDB Open Source, ScyllaDB docs, ScyllaDB documentation, ScyllaDB Documentation
+   :title: ScyllaDB Documentation
+   :description: ScyllaDB  Documentation
+   :keywords: ScyllaDB docs, ScyllaDB documentation, ScyllaDB Documentation
 
 About This User Guide
 -----------------------
 
-ScyllaDB is a distributed NoSQL wide-column database for data-intensive apps that require 
+This user guide covers topics related to ScyllaDB - a distributed NoSQL
+wide-column database for data-intensive apps that require 
 high performance and low latency.
 
-This user guide covers topics related to ScyllaDB Open Source - an open-source project that allows you to evaluate 
-experimental features, review the `source code <https://github.com/scylladb/scylladb>`_, and add your contributions
```

Comment:
You removed the link to the source code, but it's still available. I don't know if this link was needed in this specific place or not.

Reply:
I see @denesb already merged this PR so we don't need to discuss this now, but it's worthy a discussion (CC @mykaul) whether or not the main page should mention the source code or not.
In that past we were very proud of the fact you can see the source code, and even keep a backup of it in case ScyllaDB the company goes under and keeps you hanging. But maybe now that we backed away from the "open source" thing, we no longer think this.